R/Report_KRI.R

```diff
@@ -36,35 +37,44 @@
 #' )
 #'
 #' # Create Mapped Data
-#' lMapped <- RunWorkflow(lWorkflow = wf_mapping, lData = lRaw)$lData
+#' lMapped <- RunWorkflow(lWorkflow = wf_mapping, lData = lRaw)$mapping$lResults
```

Comment:
This reflects the `RunWorkflow()` changes that we've reverted. Can we make sure to revert these as well? There are also references to `$lResults` throughout other scripts and header examples. Thanks!!